administrator/language/de-DE/de-DE.ini

```diff
@@ -205,6 +205,8 @@ JFIELD_BASIS_LOGOUT_DESCRIPTION_LABEL="Beschreibungstext"
 JFIELD_BASIS_LOGOUT_DESCRIPTION_SHOW_DESC="Zeigen oder Verbergen der Abmeldebeschreibung"
 JFIELD_BASIS_LOGOUT_DESCRIPTION_SHOW_LABEL="Abmeldebeschreibung"
 JFIELD_CATEGORY_DESC="Die Kategorie, die diesen Beitrag zugeordnet ist."
+JFIELD_DISPLAY_READONLY_DESC="Ob das Feld auf Formularen angezeigt werden soll, wenn es schreibgeschützt ist. Standardmäßig wird der in der Feldgruppe eingestellte Wert übernommen."
```

Comment:
Gibt an, ob das Feld in Formularen schreibgeschützt angezeigt werden soll. Übernimmt standardmäßig den in der Feldgruppe festgelegten Wert.